tests/acceptance/volta_pin.rs

```diff
@@ -203,9 +205,7 @@ fn pin_node() {
 
     assert_that!(
         s.volta("pin node@6"),
-        execs()
-            .with_status(ExitCode::Success as i32)
-            .with_stdout_contains("[..]pinned node@6.19.62 (with npm@3.10.1066) in package.json")
```

Comment:
Seems like by default we should still have some kind of feedback to the user about which version was pinned, without requiring that they set VOLTA_LOGLEVEL to get that info.

Reply:
We do by default in a TTY (but the tests are run with subcommands, so it detects that we aren't a TTY). I could see a different approach where we always write that information out, but by default if we're running as a script then the output is limited to only errors.

Reply:
ah, gotcha - went back and read through that